source/includes/_users.md

```diff
@@ -300,3 +300,153 @@ Delete a user.
 
 ### Response
 A `User` resource.
+
+
+
+
+## Set user password
+
+> Request Example
+
+```shell
+$ curl -X POST -H "Content-Type: application/json" -H "tenant: <tenant-name>" -u user:password -d '{"password": <new-password>}' "http://<manager-ip>/api/v3/users/<user-name>"
+```
+
+```python
+# Python Client-
+client.users.set_password(<user-name>, <new-password>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "password": <new-password>,
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/users/<user-name>",
+  "method": "POST",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "username": "user",
+    "last_login_at": "2017-01-22T15:09:33.799Z",
+    "role": "user",
+    "groups": [],
+    "active": true,
+    "tenants": ["default_tenant"]
+}
+```
+
+`POST -d '{"password": <new-password>}' '"{manager-ip}/api/v3/users/{user-name}"`
+
+Delete a user.
+
+### URI Parameters
+* `user-name`: The name of the user to change his password.
+
+### Request Body
+
+Property | Type | Description
+--------- | ------- | -----------
+`password` | string | The user new password.
+
+### Response
+A `User` resource.
+
+
+
+## Set user role
+
+> Request Example
+
+```shell
+$ curl -X POST -H "Content-Type: application/json" -H "tenant: <tenant-name>" -u user:password -d '{"role": <user-role>}' "http://<manager-ip>/api/v3/users/<user-name>"
+```
+
+```python
+# Python Client-
+client.users.set_role(<user-name>, <new-role>)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64'),
+   'tenant': <tenant-name>
+}
+
+vat data = {
+    "role": <user-role>,
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/users/<user-name>",
+  "method": "DELETE",
+  "headers": headers,
+  "data": data
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example
+
+```json
+{
+    "username": "user",
+    "last_login_at": "2017-01-22T15:09:33.799Z",
+    "role": "user",
+    "groups": [],
+    "active": true,
+    "tenants": ["default_tenant"]
+}
+```
+
+`POST -d '{"role": <role>}' '"{manager-ip}/api/v3/users/{user-name}"`
+
+Set a new role for the user (`user`, `administrator`, `suspended`).
+
+* `user` - The default user.
+
+* `administrator` - Can execute Cloudify management commands (handle users, tenants, etc)
+
+* `suspended` - Exiting user that won't have access to Cloudify, without deleting it.
```

Comment:
~~Exiting user that won't have~~ **Prevents user** access to Cloudify, without deleting ~~it.~~ **them.**